### ISSUE TITLE
Formatting

### ISSUE THREAD

**Issue by burner1024** [F3mic/F3mic.github.io#2]
I suppose it shouldn't look like this
![Captura de pantalla de 2022-06-24 17-38-18](https://user-images.githubusercontent.com/1279852/175518601-f46bd348-8e86-40ec-9688-02e7486c5f57.png)

Probably should style images so that they fit and there's no horizontal scroll?

**Comment by F3mic** [comment]
Hi burn,

long time no see.
Yes, you're right. This isn't supposed to happen.
Many people stated this problem already.
I'm working on it but I lack the time and the skill to resolve this problem.
I've already found someone who could fix this but it's prio low.

Kind regards.

**Comment by F3mic** [comment]
Hi burn,

the overlapping sidebar has been fixed.
Please feel free to check.

Regards

**Comment by burner1024** [comment]
Can't see any difference, unfortunately (chromium).

**Comment by F3mic** [comment]
You're right. It's still overlapping. The only thing that has been fixed is the sidebar disappering when the browser window gets smaller than ~70rem and reappears when the browser window get bigger again.
I'm furhter investigating the overlapping issue.

**Comment by F3mic** [comment]
Ok, should work now.
It's not pretty but it works as far as I can see.
Please check.

Regards

**Comment by burner1024** [comment]
Yes, it no longer overlaps.
Another issue is that impossible to see the lower items in the sidebar. Anything past "Test your script" is never on screen.
![Captura desde 2023-05-05 06-14-14](https://user-images.githubusercontent.com/1279852/236349504-cb5fefcc-52b4-4058-80cc-8f4f8dfa90a0.png)

**Comment by F3mic** [comment]
My bad. I positioned the sidebar higher now. If the elements are still cut I need to investigate further.

**Comment by F3mic** [comment]
I see the sidebar doesn't count correctly. Firefox does it correctly but Chromium Browser don't. For example 'Mapping' should be Number 3. I'm on it.

**Comment by F3mic** [comment]
Sidebar counting has been fixed.

**Comment by F3mic** [comment]
The sidebar doesn't scroll if text overflows. I'm investigating.

**Comment by F3mic** [comment]
Sidebar is fixed now. Can be scrolled when text overflows.
